src/test/java/core/basesyntax/io/FileReaderImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.io;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FileReaderImplTest {
+    private static final String EXISTING_FILE_PATH = "src/test/resources/testInput.csv";
+    private static final String EMPTY_FILE_PATH = "src/test/resources/emptyFile.csv";
+    private static final String INVALID_FORMAT_FILE_PATH
+            = "src/test/resources/invalidFormatFile.csv";
+    private FileReader fileReader;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void read_existingFile_ok() {
+        List<String> lines = fileReader.read(EXISTING_FILE_PATH);
+        assertEquals(6, lines.size());
+    }
+
+    @Test
+    public void shouldReturnEmptyListWhenFileIsEmpty() throws IOException {
```

Comment:
use unify format of the method names (check all test cases): 

Remember about the naming of test methods.
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`. For example, if we are testing the method `register` with a `null` user's age the test method name should be `register_nullAge_notOk`. notOk is because the test expects the register method to throw an exception.

Reply:
I use the template should...when...